aws/v9.3.4/README.md

```diff
@@ -0,0 +1,52 @@
+# :zap: Giant Swarm Release v9.3.4 for AWS :zap:
+
+This release updates managed apps to the latest releases.
```

Comment:
```suggestion
This release updates managed apps to the latest releases. It also includes security fixes for 3 CVEs.
```

Reply:
I've changed it in the [other PR](https://github.com/giantswarm/releases/pull/344/files#diff-46bc0c7a2ace3de49a04a338c907e701 just now. BTW should I merge that other PR here to have less confusion? @sslavic

Reply:
Other PR got merged.